src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1880,19 +1938,7 @@ def get_user_list(self, zoneName: str):
             user_list = out
         except SubprocessError as error:
             raise DashboardException(error, http_status_code=500, component='rgw')
-
-        if len(user_list) > 0:
-            for user_name in user_list:
-                rgw_user_info_cmd = ['user', 'info', '--uid', user_name, '--rgw-zone', zoneName]
-                try:
-                    exit_code, out, _ = mgr.send_rgwadmin_command(rgw_user_info_cmd)
-                    if exit_code > 0:
-                        raise DashboardException('Unable to get user info',
-                                                 http_status_code=500, component='rgw')
-                    all_users_info.append(out)
-                except SubprocessError as error:
-                    raise DashboardException(error, http_status_code=500, component='rgw')
-        return all_users_info
```

Comment:
why this deletion ?

Reply:
This method is not used anywhere else in the code. As this was lying unused and we needed a similar method now for our use where we just need the users list and not the user info along with it, so decided to reuse this method itself.